src/drawer/__tests__/drawer-select.scenario.js

```diff
@@ -7,16 +7,18 @@ LICENSE file in the root directory of this source tree.
 // @flow
 
 import * as React from 'react';
-
+import {Button} from '../../button/index.js';
 import {StatefulSelect} from '../../select/index.js';
 import {Drawer} from '../index.js';
 
 export const name = 'drawer-select';
 
 const Example = () => {
+  const [isOpen, setIsOpen] = React.useState(true);
   return (
     <div>
-      <Drawer closeable={false} isOpen>
```

Comment:
The drawer was not even closable so the check for it to stay open when an option from select's dropdown clicked didn't test anything.